sklearn/model_selection/_validation.py

```diff
@@ -567,58 +563,33 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     return ret
 
 
-def _score(estimator, X_test, y_test, scorer, is_multimetric=False):
-    """Compute the score(s) of an estimator on a given test set.
-
-    Will return a single float if is_multimetric is False and a dict of floats,
-    if is_multimetric is True
-    """
-    if is_multimetric:
-        return _multimetric_score(estimator, X_test, y_test, scorer)
+def _score(estimator, X_test, y_test, scorer):
+    """Compute the score(s) of an estimator on a given test set."""
```

Comment:
Let's keep the comment about what is returned.

IIUC a dict is returned iff `scorer` is a dict?